src/pretix/static/pretixbase/js/asynctask.js

```diff
@@ -218,10 +218,15 @@ $(function () {
             'this page and try again.'
         ));
 
-
-        console.log($(this).get(0))
         var formData = new FormData($(this).get(0))
         formData.append('ajax', '1');
+        if (e.originalEvent.submitter) {
+            // Not supported on IE, may lead to wrong results, but we don't support IE in the backend anymore
+            var submitter = e.originalEvent.submitter;
+            if (submitter.name) {
```

Comment:
Sorry, I forgot we need to check if submitter itself is not null.

```suggestion
            if (submitter && submitter.name) {
```

Reply:
How could I not see this!? Probably too late :)